data/EventRow.css

```diff
@@ -1,10 +1,14 @@
 .event {
-    background: alpha(@colorAccent, 0.15);
-    border-radius: 3px;
-    color: shade(@colorAccent, 0.65);
-    padding: 6px;
+    background-color: alpha(@eventcolorAccent, 0.15);
+    color: shade(@eventcolorAccent, 0.65);
+    border-radius: 4px;
+    padding: 10px;
+    box-shadow: 0 1px 2px alpha (#000, 0.2),
+                0 1px 3px alpha (@eventcolorAccent, 0.15);
```

Comment:
These styles match what's in Calendar, so I'd prefer not to change this

Reply:
At least the padding should be added otherwise the text and icons will hug the border of the box